diff --git a/data.cpp b/data.cpp
index 09f6d7d..f7a3515 100755
--- a/data.cpp
+++ b/data.cpp
@@ -746,7 +746,7 @@ void DataManager::SetDefaultValues()
 	string Lun_File_str = CUSTOM_LUN_FILE;
 	size_t found = Lun_File_str.find("%");
 	if (found != string::npos) {
-		sprintf(lun_file, CUSTOM_LUN_FILE, 0);
+		sprintf(lun_file, CUSTOM_LUN_FILE);
 		Lun_File_str = lun_file;
 	}
 	if (!TWFunc::Path_Exists(Lun_File_str)) {
diff --git a/partitionmanager.cpp b/partitionmanager.cpp
index cfadbd8..e36fc4c 100755
--- a/partitionmanager.cpp
+++ b/partitionmanager.cpp
@@ -2779,7 +2779,7 @@ TWPartitionManager::usb_storage_enable (void)
   size_t found = Lun_File_str.find ("%");
   if (found != string::npos)
     {
-      sprintf (lun_file, CUSTOM_LUN_FILE, 1);
+      sprintf (lun_file, CUSTOM_LUN_FILE);
       if (TWFunc::Path_Exists (lun_file))
 	has_multiple_lun = true;
     }
@@ -2788,7 +2788,7 @@ TWPartitionManager::usb_storage_enable (void)
     {
       LOGINFO
 	("Device doesn't have multiple lun files, mount current storage\n");
-      sprintf (lun_file, CUSTOM_LUN_FILE, 0);
+      sprintf (lun_file, CUSTOM_LUN_FILE);
       if (TWFunc::Get_Root_Path (DataManager::GetCurrentStoragePath ()) ==
 	  "/data")
 	{
@@ -2818,7 +2818,7 @@ TWPartitionManager::usb_storage_enable (void)
       LOGINFO ("Device has multiple lun files\n");
       TWPartition *Mount1;
       TWPartition *Mount2;
-      sprintf (lun_file, CUSTOM_LUN_FILE, 0);
+      sprintf (lun_file, CUSTOM_LUN_FILE);
       Mount1 = Find_Next_Storage ("", true);
       if (Mount1)
 	{
@@ -2826,7 +2826,7 @@ TWPartitionManager::usb_storage_enable (void)
 	    {
 	      goto error_handle;
 	    }
-	  sprintf (lun_file, CUSTOM_LUN_FILE, 1);
+	  sprintf (lun_file, CUSTOM_LUN_FILE);
 	  Mount2 = Find_Next_Storage (Mount1->Mount_Point, true);
 	  if (Mount2 && Mount2->Mount_Point != Mount1->Mount_Point)
 	    {
@@ -2860,7 +2860,7 @@ TWPartitionManager::usb_storage_disable (void)
 
   for (index = 0; index < 2; index++)
     {
-      sprintf (lun_file, CUSTOM_LUN_FILE, index);
+      sprintf (lun_file, CUSTOM_LUN_FILE);
       ret = TWFunc::write_to_file (lun_file, str);
       if (ret < 0)
 	{
